Potatso/AppInitializer.swift

```diff
@@ -56,9 +56,9 @@ class AppInitializer: NSObject, AppLifeCycleProtocol {
         #endif
     }
 
-    func configHelpShift() {
-        HelpshiftCore.initialize(with: HelpshiftAll.sharedInstance())
-        HelpshiftCore.install(forApiKey: HELPSHIFT_KEY, domainName: HELPSHIFT_DOMAIN, appID: HELPSHIFT_ID)
-    }
+//    func configHelpShift() {
```

Comment:
I thought about removing HelpShift as well. I will see how we will proceed this i.e cherry pick commits, or else.